source/_integrations/hue.markdown

```diff
@@ -36,20 +36,37 @@ There is currently support for the following device types within Home Assistant:
 
 ## Lights for Hue zones and rooms
 
-The Hue concept is based on Rooms and zones. Although the underlying Hue lights are exposed directly to Home Assistant it might also be useful to interact with the `grouped lights` of the Hue ecosystem, for example to turn all lights in a Hue group on/off at the same time.
+The Hue concept is based on Rooms and Zones. Although the underlying Hue lights are exposed directly to Home Assistant it might also be useful to interact with the `grouped lights` of the Hue ecosystem, for example to turn all lights in a Hue group on/off at the same time.
 
 Home Assistant creates lights for each Hue zone/room automatically but disables them by default.
 If you'd like to use those `grouped lights`, you can enable them from Configuration --> Integrations --> Hue --> entities.
 
 ## Scenes
 
-In the Hue concept you can create (dynamic) scenes for the lights within rooms and zones. These Hue scenes are automatically imported in Home Assistant and available as scene entities. Creating or editing Hue scenes in Home Assistant is not supported.
+In the Hue concept you can create (dynamic) scenes for the lights within rooms and zones. You can create, edit and delete Hue scenes from the (official) Hue app on iOS and Android. Each Zone/Room can have it's own scenes assigned and there is a large library of precreated scenes for specific moods. These Hue scenes are automatically imported in Home Assistant and they're available as `scene entities`. Creating or editing Hue scenes in Home Assistant is not supported.
 
+`It is advised to use Hue scenes for controlling multiple lights at once for a smooth experience. If you individually control multiple lights and/or use Home Assistant scenes, each light command will be sent to each light one by one which doesn't give a very good user experience, while using a Hue scene sends commands to all lights at once in an optimized way, resulting in a smooth experience.`
+
+### Service `hue.activate_scene`
+
+To have more control over Hue scenes we've implemented a secondary, more advanced service to activate a Hue scene and set some properties at the same time, such as the Dynamic mode and/or brightness.
+
+| Service Data Attribute | Required | Description                                                                                   |
+| ---------------------- | -------- | --------------------------------------------------------------------------------------------- |
+| `entity_id`            | yes      | Entity ID of the Hue Scene entity you want to activate.                                       |
```

Comment:
This is an entity service, so entity_id would be optional in that case (as a area or device can be targeted instead)

Reply:
What text edit do you suggest ? Just changing the required to "no"

Reply:
I guess, that is how this is generally reflected in our documentation.

Having more support for `target` in our documentation is still an (open) more general task that we need to solve globally.

Reply:
Yeah, I checked some other integration docs with entity services and this (entity_id as required) is how they do it so I just shamelessly copied it.

Reply:
Shall I change it to "target (any of entity, device or area)" ?

Reply:
@marcelveldt Yeah is it fine by me...
Lets do that.

I should really create some time to finish up importing services from the core repository to render docs 😬